js/main.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+var NUMBER_ADD = 8;
+var LOCATION_Y_MIN = 130;
+var LOCATION_Y_MAX = 630;
+var fragment = document.createDocumentFragment();
+var adUnit = document.querySelector('#pin').content.querySelector('.map__pin');
+var adUnitImg = adUnit.querySelector('img');
+var insertionUnit = document.querySelector('.map__pins');
+
+document.querySelector('.map').classList.remove('map--faded');
+
+/* тип жилья
+var getTypeHousing = function () {
+  var TypeHousing = offer.type[Math.floor(Math.random() * (offer.type.length + 1))];
+  return TypeHousing;
+};
+
+/* массив строк случайной длины, содержащий адреса фотографий
+var getRandomLength = function () {
+  offer.photos.length = Math.floor(Math.random() * (offer.photos.length + 1));
+  return offer.photos.length;
+};
+
+var offer = {
+  'title': 'заголовок',
+  'address': '600, 350',
+  'price': 1000,
+  'type': ['palace', 'flat', 'house', 'bungalo'],
+  'rooms': 'число, количество комнат',
+  'guests': 'число, количество гостей, которое можно разместить',
+  'checkin': '12:00, 13:00,14:00',
+  'checkout': '12:00, 13:00,14:00',
+  'features': ['wifi', 'dishwasher', 'parking', 'washer', 'elevator', 'conditioner'],
+  'description': 'строка с описанием',
+  'photos': ['http://o0.github.io/assets/images/tokyo/hotel1.jpg',
+    'http://o0.github.io/assets/images/tokyo/hotel2.jpg', 'http://o0.github.io/assets/images/tokyo/hotel3.jpg'
+  ]
+};
+*/
+
+/* случайное число, координата x метки на карте. Значение ограничено размерами блока, в котором перетаскивается метка
+var obz = document.querySelector('.map__overlay').style.width = '100%';
+не совсем понимаю, что использовать */
+
+var locationXY = function () {
```

Comment:
```suggestion
var locateXY = function () {
```

location - это существительное, а https://up.htmlacademy.ru/javascript/19/criteries#b6